assets/scss/components/_waiting.scss

```diff
@@ -0,0 +1,19 @@
+/*
+Waiting State
+
+Markup:
+<span class="waiting"></span>
+
+Styleguide Waiting State
```

Comment:
As opposed to? It's part of a component.

You're raising a bigger issue around the organising of components that consist of both styles *and* js, but that's not for this PR.